### PR TITLE
Add cx to bitcoinDashboard.js

### PR DESCRIPTION
**Pull request by luixxiul**
Addresses #8126

Also:
- Add testIds
- Replace styles.bitcoinAddressButton with globalStyles.spacing.dialogInsideMargin

Auditors:

Test Plan:
1. Open about:preferences#payments
2. Click "Add funds..."
3. Make sure every button works

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:

### CODE REVIEW

File: app/renderer/components/preferences/payment/bitcoinDashboard.js [hunk 1]
```diff
@@ -156,27 +175,46 @@ class BitcoinDashboard extends ImmutableComponent {
...
   }
 
   panelFooter () {
     if (this.props.ledgerData.get('buyURLFrame')) {
       return <div className={css(paymentCommon.panelFooter, styles.panelFooter)}>
-        <Button l10nId='done' className={css(commonStyles.whiteButton)} onClick={this.props.hideParentOverlay} />
+        {/* TODO: refactor button.js */}
+        <Button className='whiteButton'
+          l10nId='done'
+          testId='done'
```
Comment (NejcZdovc) [inline app/renderer/components/preferences/payment/bitcoinDashboard.js]: Can we change this to more specific id?

File: app/renderer/components/preferences/payment/bitcoinDashboard.js [hunk 2]
```diff
@@ -156,27 +175,46 @@ class BitcoinDashboard extends ImmutableComponent {
...
       return <div className={css(paymentCommon.panelFooter, styles.panelFooter, styles.coinbaseFooter)}>
         <div className={css(styles.coinbase)}>
           <div className={css(styles.coinbaseLogo)} />
           <span className={css(styles.coinbaseMessage)} data-l10n-id='coinbaseMessage' />
         </div>
-        <Button l10nId='done' className={css(commonStyles.whiteButton)} onClick={this.props.hideParentOverlay} />
+        {/* TODO: refactor button.js */}
+        <Button className='whiteButton'
+          l10nId='done'
+          testId='done'
```
Comment (NejcZdovc) [inline app/renderer/components/preferences/payment/bitcoinDashboard.js]: ditto

File: app/renderer/components/preferences/payment/bitcoinDashboard.js [hunk 3]
```diff
@@ -156,27 +175,46 @@ class BitcoinDashboard extends ImmutableComponent {
...
+          onClick={this.props.hideParentOverlay} />
       </div>
     } else {
       return <div className={css(paymentCommon.panelFooter, styles.panelFooter)}>
-        <Button l10nId='done' className={css(commonStyles.whiteButton)} onClick={this.props.hideParentOverlay} />
+        {/* TODO: refactor button.js */}
+        <Button className='whiteButton'
+          l10nId='done'
+          testId='done'
```
Comment (NejcZdovc) [inline app/renderer/components/preferences/payment/bitcoinDashboard.js]: ditto

**Comment by luixxiul** [comment]
@NejcZdovc done

**Pull request merged**